npm-react-codemod/transforms/class.js

```diff
@@ -452,7 +452,7 @@ function updateReactCreateClassToES6(file, api, options) {
       path = j(classPath).closest(j.VariableDeclaration);
     }
 
-    path.replaceWith(
+    j(path.get()).replaceWith(
```

Comment:
You shouldn't have to do this that's more likely a bug in jscodeshift actually. I will investigate.